tests/README.md

```diff
@@ -114,6 +113,17 @@ $ cd <root directory of cloudify-manager repository>
 $ pytest -s tests/integration_tests/tests/agentless_tests/test_workflow.py:BasicWorkflowsTest.test_execute_operation
 ```
 
+### Saving the Cloudify Manager's logs
+In order to save the logs for tests, specify the path via an environment variable as follows:
+
+`export CFY_LOGS_PATH=<YOUR-PATH-HERE>`
```

Comment:
Maybe change CFY_LOGS_PATH => to CFY_LOGS_PATH_REMOTE

And add example of how the path should look like when in remote mode

Reply:
:+1: